haiku-apps/projectconceptor/projectconceptor-0.1.1.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="A Haiku diagramming application like Microsoft Visio"
+DESCRIPTION="ProjectConceptor is a modular application for Haiku and BeOS \
```

Comment:
The "Haiku" (and BeOS) is unnecessary, as it's pretty much implied, because it's a package at HaikuDepot.
Maybe add the key features of ProjectConceptor instead. That way it becomes more attractive for a possible user and it'll appear more easily in full text searches as there's a higher possibility of search terms getting hit.